### PR TITLE
Add marketing material to SvelteKit homepage

### PR DESCRIPTION
**Pull request by benmccann**
The SvelteKit homepage is a little empty at the moment. We should describe why people should use SvelteKit

### CODE REVIEW

File: sites/kit.svelte.dev/src/routes/index.svelte [hunk 1]
```diff
@@ -73,8 +117,7 @@
 		</p>
 
 		<p>
-			Read the <a href="https://svelte.dev/blog/whats-the-deal-with-sveltekit">introductory blog</a>
-			post to learn more.
+			<a href="https://stackblitz.com/fork/svelte-kit">Try on StackBlitz</a> or create a project locally.
```
Comment (dominikg) [inline sites/kit.svelte.dev/src/routes/index.svelte:120]: node.new/sveltekit instead?

Comment (benmccann) [reply sites/kit.svelte.dev/src/routes/index.svelte]: done. what's that URL though? I don't know who owns it...

Comment (dominikg) [reply sites/kit.svelte.dev/src/routes/index.svelte]: official launcher by stackblitz afaik. https://mobile.twitter.com/Rich_Harris/status/1461876128474447875

**Review by arxpoetica — APPROVED** [review]
Nice!

**Pull request merged**